packages/block-editor/src/components/block-variation-transforms/index.js

```diff
@@ -35,8 +35,7 @@ function VariationsButtons( {
 			</VisuallyHidden>
 			{ variations.map( ( variation ) => (
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
| Before | After     |
| :-------- | :------- |
|  ![block-variation-transform-before](https://github.com/user-attachments/assets/888d5d04-bee8-408d-bfb2-bd1e3300ade1) | ![block-variation-transform-after](https://github.com/user-attachments/assets/181f0f6b-a3b8-4b1d-84c9-d165a827c7a6) |

Reply:
I'm not sure about this one. I think it should be more like [ToggleGroupControl](https://wordpress.github.io/gutenberg/?path=/docs/components-experimental-togglegroupcontrol--docs). IE the outer container is `40px`, while the inner buttons are `32px` (`compact` size).

Reply:
@jameskoster what about just using a ToggleGroupControl?

Reply:
The visible stroke in the resting state would create additional weight that is probably unwanted in this UI. 

We have an issue to explore the `ToggleGroupControl` design [here](https://github.com/WordPress/gutenberg/issues/64439), which might make it appropriate for use in its vanilla state if implemented.

But until then the simpler approach is probably to just make these buttons `compact` (32px).

Reply:
<img width="355" alt="image" src="https://github.com/user-attachments/assets/4b5d3aa8-5f5a-49b4-9e6f-5a9b6bf5deb1">

I have updated it to use compact size.